uqcsbot/scripts/dice.py

```diff
@@ -0,0 +1,31 @@
+from random import choice
+from uqcsbot import bot, Command
+
+def is_number(message):
```

Comment:
Python strings have a builtin str.isnumeric() method that can replace this whole method.

Reply:
This was copied from `mock.py`. (`isnumeric()` returns `False` if string contains a `.`.)  
Removed.